css/index.css

```diff
@@ -1,3 +1,375 @@
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+main .top-content {
+  padding: 0 10px;
+	display: flex;
+}
+
+main .top-content .text-content {
+  width: 710px;
+
+  margin-right: 34px;
+}
+
+main .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+main .middle-img {
+  margin-top: 45px;
+}
+
+main .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+	display: flex;
+}
+
+main .bottom-content .text-content {
+  width: 275px;
+
+  margin-right: 12px;
+}
+
+main .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 100px 0 30px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/*   Services     */
+
+
+.services .services-header-img{
+	display: block;
+	margin: 30px 0;
+}
+.services .services-info{
+	padding-bottom: 20px;
+	border-bottom: 1px solid black;
+}
+.services .services-info h1{
+		font-size: 25px;
+		font-weight: bold;
+}
+
+ /*       widgets     */
+.services .widgets{
+	height: auto;
+	display: flex;
+	flex-wrap: wrap;
+	margin-top: 35px;
+
+	justify-content: space-between;
+	align-items:  space-between;
+	align-content: space-between;
+	border-bottom: 1px solid black;
+}
+.services .widget-box{
+	width: 420px;
+	height: 250px;
+	background: #F4F3F4;
+	border: 1px solid black;
+	margin-bottom: 40px;
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+}
+.services .widget-box h2,.services .widget-box p{
+	margin: 25px 10px 0 20px;
+}
+.services .button{
+	background: white;
+	border: 1px solid gray;
+	text-align: center;
+	width: 120px;
+	height: 35px;
+	padding: 10px;
+	font-size: 12px;
+	border-radius: 5px;
+}
+
+
+/*     foooter    */
+.services .footer-info{
+	display: flex;
+	margin-top: 40px;
+}
+.services .footer-info section{
+	margin-right: 50px;
+
+}
+.services .footer-info li{
+	line-height: 2em;
+	list-style-type: square;
+  list-style-position: inside;
+}
+
+
+/*        responsive            */
+
+@media (max-width:800px) {
+  
+    .container {
+      width: 720px;
+    }
+    header{
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        
+
+    }
+    header nav{
+        order:1;
+        width: 100%
+    }
+    
+    header nav a {
+      width: 100%;
+      
+    
+      
+      display: block;
+      text-align: center;
+    }
+    .cta{
+        justify-content: center;
+        margin-top: 0;
+    }
+    .cta img{
+        display: none;
+    }
+    .cta .cta-text{
+    align-self: center;
+    margin-top: 80px;
+    }
+
+    .middle-img{
+        width: 100%
+        
+    }
+  
+  }
+  
+  /* mobile */
+  @media (max-width:500px) {
+    .container{
+        width: 100%;
+     }
+    header nav {
+        flex-direction: column;
+        margin: 20px;
+        align-items: center;
+        border-top: 1px solid gray;
+      }
```

Comment:
You have some spacing inconsistencies throughout your CSS. Try using the prettier extension (https://prettier.io/) if you don't want to make it uniform by hand!